tests/main/snap-logs/task.yaml

```diff
@@ -16,7 +16,7 @@ execute: |
     snap logs test-snapd-service.test-snapd-other-service | MATCH "running"
 
     echo "check output lines for the logs"
-    snap logs -n=1 test-snapd-service | MATCH "running"
+    snap logs -n=1 test-snapd-service | MATCH "systemd|running"
```

Comment:
Should we simply check for more lines than one?

Reply:
@mvo5, the test is currently checking with -n=all and -n=1, the problem is that looking at the logs sometimes we get about  5~10 consecutive lines for the systemd unit. If it is ok for you I could update this to check with -n=20 and I think that should be enough to prevent any error.